DigitalLearningSolutions.Data/ApiClients/LearningHubApiClient.cs

```diff
@@ -103,5 +108,19 @@ private static string GetQueryString(string key, string? value)
         {
             return string.IsNullOrEmpty(value) ? "" : $"{key}={value}";
         }
+
+        private async Task<string?> GetAsyncAndProcessResult(string requestUri)

```

Comment:
Name's fine, though it doesn't really give any more information than `GetStringAsync` except that it "does something extra ㊙️ " - I wouldn't have a problem with you calling it `GetStringAsync`.